### ISSUE TITLE
Pass CI check not updating

### ISSUE THREAD

**Issue by Feng1231** [nus-cs2103-AY2425S1/forum#381]
Our team enabled the pass CI check before allowing pull requests to be approved.
However, that status is not updated even after all checks passed.

Does anyone knows how to solve this?
![image](https://github.com/user-attachments/assets/98fce3dc-30e9-496b-8f22-b199e8c130dc)

**Comment by damithc** [comment]
@Feng1231 Hmm... that's odd. Could be an issue on the GitHub side. It's not urgent to resolve though. You can triggering CI again to see if it is a temporary issue. 
An admin should still be able to merge the PR even if all conditions are not met.

**Comment by yashma-sonara** [comment]
Hi, dropping in from your senior batch!

Is this still an issue. I took a look at your github repo, and all commits and the past pr's look to pass CI now?

**Comment by Feng1231** [comment]
>@Feng1231 Hmm... that's odd. Could be an issue on the GitHub side. It's not urgent to resolve though. You can triggering CI again to see if it is a temporary issue.

We have tried
1. Creating PRs from each of our member's fork. None of it could be merged until that 'pass CI' check is done (in which it's never updated, even after ~5-10 mins of wait)
2. Pushing new commits to our forks after creating PRs, triggering a re-checking of the Java-CI and code cov checks. Even so, after the new round of check, the 'pass CI' check still won't update.
 
> Hi, dropping in from your senior batch!
> 
> Is this still an issue. I took a look at your github repo, and all commits and the past pr's look to pass CI now?

It was an issue. but we disabled the 'pass CI' condition in the repo settings, and so this 
![image](https://github.com/user-attachments/assets/c5144edd-d357-45c9-b780-cc5d4bb34535)
disappeared from the PR.
